packages/vscode-ui5-language-assistant/test/manual-tests/annotation_metaPath_attribute_handling.md

```diff
@@ -0,0 +1,72 @@
+# Visual Studio Code - FPM: macros elements metaPath completion and diagnostics (annotation path)
+
+## Associated user stories:
+
+[#505](https://github.com/SAP/ui5-language-assistant/issues/505) Provide code completion and diagnostics for contentPath and metaPath values
+
+## Install latest UI5 Language Assistant
+
+[UI5 Language Assistant](https://marketplace.visualstudio.com/items?itemName=SAPOSS.vscode-ui5-language-assistant)
+
+## Test project
+
+1. Prepare an UI5 Project
+   - Your project or
+   - Clone [ui5-language-assistant](https://github.com/SAP/ui5-language-assistant) repository and open `test-packages/framework/projects/cap`
+2. Launch VSCode and open project root folder
+3. Install project dependencies using command `npm install`
+
+**Note:** For this manual test please use `test-packages/framework/projects/cap` project
+
+### **Step 1**: Code completion for metaPath
+
+1. Open annotations file `app\manage_travels\annotations.cds` and append following snippet to the end of file and save it:
+
+```cpp
+annotate service.Travel with @(
+    UI.Chart #sample1: {
+        type: '',
```

Comment:
I guess `$Type` is cds syntax

Reply:
No, it is chart type, e.g. Line, Bar, etc.. 
For this test it doesn't matter, just mandatory property. Provided to avoid CDS lsp complaints